### PR TITLE
Add enableAutomaticUpgrade property to VMExtensions for VMs and VMSS

### PR DESCRIPTION
**Pull request by chjaress**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json [hunk 1]
```diff
@@ -7260,6 +7260,10 @@
           "type": "boolean",
           "description": "Indicates whether the extension should use a newer minor version if one is available at deployment time. Once deployed, however, the extension will not upgrade minor versions unless redeployed, even with this property set to true."
         },
+        "enableAutomaticUpgrade": {
```
Comment (yungezz) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json:7263]: is there default value for this property?

**Comment by yungezz** [comment]
hi @chjaress could you pls help to look at the comments?

File: specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json (hunk 1 above)
Comment (chjaress) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json:7263]: Hi @yungezz the default value is null for this property.

**Review by yungezz — APPROVED** [review]

**Review by yungezz — APPROVED** [review]

**Comment by yungezz** [comment]
hi @chjaress is this PR ready to merge, that means service change is public at PROD.

**Comment by chjaress** [comment]
Hi @yungezz this PR is ready to merge as the feature is already in PROD

**Pull request merged**